core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -72,7 +72,6 @@ object ConsumerGroupCommand {
     } catch {
       case e: Throwable =>
         println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
```

Comment:
Sorry for the delay - was on vacation. I've made the change though I wasn't able to test it out on my environment so please do check it and let me know if it works/doesn't :)